SnowSQL/public/Invoke-SnowSql.ps1

```diff
@@ -56,16 +62,16 @@ function Invoke-SnowSql
         try
         {
             $snowSql = Get-Command snowsql -ErrorAction Stop |
-                Select-Object -First 1 -ExpandProperty Source
+            Select-Object -First 1 -ExpandProperty Source
```

Comment:
Indent please. If your VSCode is set to not indent pipeline continuations, you should fix that PowerShell extension setting. The setting default has been updated to correctly indent per powershell guidelines, however I do not know if the VSCode powershell team updated it for people that had the previous extension installed.

Regardless, pipeline continuations should be indented as the previous code was.

Reply:
Noted :)
I added ` "powershell.codeFormatting.pipelineIndentationStyle": "IncreaseIndentationAfterEveryPipeline",` in the`settings.json` and it seems to work :+1: